src/main/java/io/phasetwo/service/resource/OrganizationsResource.java

```diff
@@ -345,4 +463,20 @@ private void createOrganization(
       throw ErrorResponse.error(e.getMessage(), Response.Status.INTERNAL_SERVER_ERROR);
     }
   }
+
+  void memberFromInvitation(InvitationModel invitation, UserModel user) {
```

Comment:
Doesn't this logic already live somewhere else? Let's just make it a static so we don't duplicate.